### PR TITLE
fix npm command

### PR DESCRIPTION
**Pull request by johannesnagl**
"run" is missing i guess. see "npm run serve" above ;)

### CODE REVIEW

**Comment by mxstbr** [comment]
Only arbitrary commands have to be run with `npm run-script` (long version of `npm run`), some common, standardised commands like `npm start`, `npm test` and others can be run while omitting the `run`.

It gets even better, if you omit the `run` and enter `npm test`, npm will run the following commands in that order: `npm run pretest`,  `npm run test`, `npm run posttest`. So if you have a test script and need to do something beforehand, put it in the `pretest` script and it'll run automatically before `test`! 

See: https://docs.npmjs.com/misc/scripts

Thanks for the PR, much appreciated! Also check out the v3.0.0 branch while you're at it.

**Comment by johannesnagl** [comment]
:+1: i'm sorry! 

tl;dr of this PR:
1336405435_fat_guy_trampoline_dunk_fail

**Comment by mxstbr** [comment]
Hahaha don't worry, a big thanks for even submitting a PR!
